portafly/src/components/data-list/modals/SendEmailModal.tsx

```diff
@@ -41,36 +39,29 @@ const SendEmailModal: React.FunctionComponent<Props> = ({
         addAlert({ id: 'success', variant: 'success', title: t('toasts.send_email_start') })
       })
       .catch(() => {
-        actionFailed()
-        addAlert({ id: 'error', variant: 'danger', title: t('toasts.send_email_error') })
+        const error = t('toasts.send_email_error')
+        actionFailed(error)
       })
   }
```

Comment:
Not needed for this PR, but `onSubmit` fn are super similar, we might want to refactor in the future